scripts/xcpng-fs-diff.py

```diff
@@ -270,18 +273,23 @@ def main():
         '/etc/passwd*',
         '/etc/pki/ca-trust/extracted/java/cacerts',
         '/etc/pki/java/cacerts',
+        '/etc/resolv.conf',
         '/etc/shadow*',
         '/etc/ssh/ssh_host_*_key.pub',
         '/etc/ssh/ssh_host_*_key',
+        '/etc/stunnel/*.pem',
         '/etc/sysconfig/network',
         '/etc/sysconfig/network-scripts/interface-rename-data/*',
         '/etc/sysconfig/xencommons',
+        '/etc/sysctl.d/91-net-ipv6.conf',
         '/etc/vconsole.conf',
         '/etc/xensource-inventory',
         '/etc/xensource/boot_time_cpus',
         '/etc/xensource/ptoken',
+        '/etc/xensource/xapi-pool-tls.pem',
         '/etc/xensource/xapi-ssl.pem',
         '/opt/xensource/gpg/trustdb.gpg',
+        '*.pyc',
```

Comment:
Can we be more precise? Changes in pyc files inside /usr, for example, would indicate that there's a packaging issue.

Reply:
These files are generated at first run and are not distributed with packages, right ? So .pyc files might differ if they include some kind of generation timestamp.

Reply:
They are supposed to distributed with packages, on the contrary, so that /usr may be read-only. But there remains some which aren't. For example, the XAPI plugins in /etc, IIRC. That's why I would not ignore all of them, only the ones we know we want to ignore.